packages/appeals-service-api/src/services/back-office-v2/index.js

```diff
@@ -0,0 +1,43 @@
+const { getAppeal } = require('../appeal.service');
+const { broadcast } = require('../../data-producers/appeal-producer');
+const { isFeatureActive } = require('../../configuration/featureFlag');
+const formatters = require('./formatters');
+
+class BackOfficeV2Service {
+	constructor() {}
+
+	/**
+	 * @param {string} appealId
+	 * @returns {Promise<Array<*> | void>}
+	 */
+	async submitAppeal(appealId) {
+		const appeal = await getAppeal(appealId);
+
+		if (!appeal) throw new Error(`Appeal ${appealId} not found`);
+
+		const isBOIntegrationActive = await isFeatureActive('appeals-bo-submission', appeal.lpaCode);
+		if (!isBOIntegrationActive) return;
+
+		if (!appeal.appealType)
+			throw new Error(`Appeal type could not be determined on appeal ${appealId}`);
+
+		return await broadcast(formatters.appeal[appeal.appealType](appeal));
+	}
+
+	/**
+	 * @param {*} questionnaireResponse
+	 * @returns {Promise<Array<*> | void>}
+	 */
+	async submitQuestionnaire(questionnaireResponse) {
```

Comment:
`submitAppeal` just expects an id and is responsible for getting the data itself, but the controller where the existing `submitQuestionnaire` function was used is responsible for getting the data. Consider picking one of these paradigms and aligning these methods.

Reply:
I did this it was good